### PR TITLE
Add FW-CV Comms Code off UART interface

### PR DESCRIPTION
**Pull request by Shrinjay**
Note, we're just waiting for FW to confirm that the output from this code looks alright, then I'll update the test. We can merge this right away if we want, though.

### CODE REVIEW

File: tests/test_fltconn_send.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "data": {
+            "longitude": 1.23,
+            "latitude": 1.23,
+            "qr_scan_flag": 1,
+            "takeoff_command": 0,
+            "detect_flag": 1
+        }
+    }
+
+    print(build_fijo_bytearray(command))
```
Comment (Shrinjay) [inline tests/test_fltconn_send.py:16]: We can update this test later to have a useful assertion, right now it just tests that the code doesn't crash

File: modules/communications/FltConnSend.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    detect_flag = request['data']['detect_flag']
+    takeoff_command = request['data']['takeoff_command']
+
+    def pack_to_int(val):
+        return struct.pack("i", val)
+
+    def pack_to_float(val):
+        return struct.pack("f", val)
+
+    byte_list = [
```
Comment (Shrinjay) [inline modules/communications/FltConnSend.py:38]: Code that handles byte packing, incase you're from fw

File: modules/commsInterface/commsInterface.py [hunk 3]
```diff
@@ -66,14 +66,14 @@ def __init__(self, idVendor=None, idProduct=None):
         # use lsusb -v in the terminal to list the idVendor and idProduct of connected usb devices 
         self.idVendor = idVendor
         self.idProduct = idProduct
-        super().__init__(self, 0)
+        super().__init__(0)
```
Comment (rayjinghaolei) [inline modules/commsInterface/commsInterface.py:69]: not really a comment but isn't "self" needed for instanciating the subclass?

**Review by rayjinghaolei — COMMENTED** [review]
not very familiar with the details of it but looks good overall

**Pull request merged**